Formula/certigo.rb

```diff
@@ -0,0 +1,38 @@
+class Certigo < Formula
+  desc "Utility to examine and validate certificates in a variety of formats"
+  homepage "https://github.com/square/certigo"
+  url "https://github.com/square/certigo.git",
+    :tag => "v1.1.0",
+    :revision => "beac5ca8f48521a8361df8c953e66902f1d6632c"
+  sha256 "1ca8ee1130d57fb70b1c21cc9311d8b036b86d456eb88888d603e88fbb1d056a"
+
+  depends_on "go" => :build
+
+  def install
+    system "./build"
+    bin.install "bin/certigo"
+  end
+
+  test do
+    (testpath/"test.crt").write("-----BEGIN CERTIFICATE-----
```

Comment:
Please use `.write <<-EOS.undent`  here and then indent appropriately. Looks a bit cleaner.